content/docs/2.1/scalers/rabbitmq-queue.md

```diff
@@ -31,7 +31,7 @@ triggers:
 
 - `queueName`: Name of the queue to read message from. Required.
 - `queueLength`: Queue length target for HPA. Default is 20. Optional.
-- `protocol`: Protocol to be used for communication. Either `http` or `amqp`. It should correspond with the `host` value.
+- `protocol`: Protocol to be used for communication. Either `auto`, `http`, or `amqp`. It should correspond with the `host` value. Optional, will autodetect based on the `host` URL if possible.
```

Comment:
Could you please update the line 19, with the new behavior? (it is `Optional` and new default value. Thanks!

Reply:
Good catch! Updated.